src/solvers/refinement/string_refinement.cpp

```diff
@@ -188,7 +188,7 @@ static void display_index_set(
   for(const auto &i : index_set.cumulative)
   {
     const exprt &s=i.first;
-    stream << "IS(" << from_expr(ns, "", s) << ")=={" << eom;
+    stream << "IS(" << format(s) << ")=={" << eom;
```

Comment:
Could you please add a follow-up commit that removes the now unnecessary `ns` parameter from various functions?

Reply:
done